news/_posts/core-weekly/2019-05-07-coreweekly-week-18-2019.md

```diff
@@ -0,0 +1,206 @@
+* [#13659](https://github.com/PrestaShop/PrestaShop/pull/13659):  Adding close welcome module in tests Attribute, by [@boubkerbribri](https://github.com/boubkerbribri)
+
+
+### Install
+
+* [#13471](https://github.com/PrestaShop/PrestaShop/pull/13471): Set null values when trying to retrieve the non-existing default shop, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+
+
+## Code changes in the '1.7.5.x' branch (for 1.7.5.2)
+
+### Core
+
+* [#13580](https://github.com/PrestaShop/PrestaShop/pull/13580): Fixed dispatched hook on the Grid presenter, by [@mickaelandrieu](https://github.com/mickaelandrieu)
+* [#13599](https://github.com/PrestaShop/PrestaShop/pull/13599): Forbid URLS to be added in customer names, by [@PierreRambaud](https://github.com/PierreRambaud)
+
+
+### Tests
+
+* [#13583](https://github.com/PrestaShop/PrestaShop/pull/13583): Fix selector in E2E tests for addons catalog page and another one in the autoupgrade test. Thank you [@nesrineabdmouleh](https://github.com/nesrineabdmouleh)
+
+
+## Code changes in the '1.6.1.x' branch (for 1.6.1.24)
+
+### Core
+
+* [#13607](https://github.com/PrestaShop/PrestaShop/pull/13607): Forbid URLs to be inserted into Name fields, by [@PierreRambaud](https://github.com/PierreRambaud)
+
+
+## Code changes in modules, themes & tools
+
+### Auto upgrade
+
+* [#279](https://github.com/PrestaShop/autoupgrade/pull/279): Apply fixes from phpstan reviews, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+* [#300](https://github.com/PrestaShop/autoupgrade/pull/300): Improve report of errors and fix notices, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+* [#302](https://github.com/PrestaShop/autoupgrade/pull/302): Display error when access is not granted, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+* [#303](https://github.com/PrestaShop/autoupgrade/pull/303): Enable php-cs-fixer on the project, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+* [#305](https://github.com/PrestaShop/autoupgrade/pull/305): Add Rollback explanation in readme, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+* [#307](https://github.com/PrestaShop/autoupgrade/pull/307): Improve configuration getter for image backup, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+
+
+### Legal compliance
+
+* [#64](https://github.com/PrestaShop/ps_legalcompliance/pull/64): Unary operator spaces. Thank you [@MathiasReker](https://github.com/MathiasReker)
+
+
+### OnBoarding
+
+* [#56](https://github.com/PrestaShop/welcome/pull/56): Fix theme & logo, by [@PierreRambaud](https://github.com/PierreRambaud)
+* [#57](https://github.com/PrestaShop/welcome/pull/57): Update module version, README and LICENSE, by [@jolelievre](https://github.com/jolelievre)
+* [#58](https://github.com/PrestaShop/welcome/pull/58): Version updated to v6.0.0, by [@jolelievre](https://github.com/jolelievre)
+
+
+### Update of readme.md and license.md
```

Comment:
Title is a bit strange 🤔 maybe we can explain what this means ?

Reply:
I will improve the title.

Reply:
is it better this way ?

Reply:
Yes 😄